packages/fxa-auth-server/test/local/routes/emails.js

```diff
@@ -88,6 +198,82 @@ function runTest(route, request, assertions) {
   return route.handler(request).then(assertions);
 }
 
+// Called in /recovery_email/set_primary, however the promise is not waited for
+// so we test the function independently as it doesn't affect the route success.
+describe('update zendesk primary email', () => {
+  afterEach(() => {
+    nock.cleanAll();
+  });
+
+  it('should update the primary email address', async () => {
+    const uid = '1234-0000';
+    nock(`https://${SUBDOMAIN}.zendesk.com`)
+      .get('/api/v2/search.json')
+      .query(true)
+      .reply(200, MOCK_SEARCH_USERS_SUCESS);
+    nock(`https://${SUBDOMAIN}.zendesk.com`)
+      .get(`/api/v2/users/${ZENDESK_USER_ID}/identities.json`)
+      .reply(200, MOCK_FETCH_USER_IDENTITIES_SUCCESS);
+    nock(`https://${SUBDOMAIN}.zendesk.com`)
+      .put(`/api/v2/users/${ZENDESK_USER_ID}/identities/${IDENTITY_ID}.json`)
+      .reply(200, MOCK_UPDATE_IDENTITY_SUCCESS);
+
+    try {
+      await updateZendeskPrimaryEmail(
+        zendeskClient,
+        uid,
+        'test@example.com',
+        'updated.email@example.com'
+      );
```

Comment:
Should we check to see if those requests were made?  I don't see any assertions.